src/dateutils.ts

```diff
@@ -58,6 +58,15 @@ export function month(date: XDate) {
   return fromTo(firstDay, lastDay);
 }
 
+function weeksDays(startDate: XDate, numberOfWeeks: number) {
```

Comment:
One last thing, sorry for not mentioning this before, please add tests for the function in the dateutils.spec file